packages/experimental/Avatar/ios/AvatarViewManager.swift

```diff
@@ -15,12 +23,12 @@ class AvatarViewManager: RCTViewManager {
 	override func constantsToExport() -> [AnyHashable : Any]! {
 		return [
 			"sizes" : [
-				"xSmall" : AvatarSize.extraSmall.size.width,
-				"small" : AvatarSize.small.size.width,
-				"medium" : AvatarSize.medium.size.width,
-				"large" : AvatarSize.large.size.width,
-				"xLarge" : AvatarSize.extraLarge.size.width,
-				"xxLarge" : AvatarSize.extraExtraLarge.size.width
+				"xSmall" : 16,
```

Comment:
The way it is now, FURN And FUA sizes can get out of sync which might be something worth avoiding if we can pull these directly from FUA?

Reply:
The SwiftUI Avatar sizes gets its' size values from the token pipeline inside FUA. We've had the question before of how we reconcile the setting of tokens between FURN and FUA in the case of wrapped FUA views. The direction I've been going towards is to have the source of truth be the token pipeline inside FURN, as ideally they should match. On that note, something I could investigate next is to get those tokens into FURN so that I don't have to hardcode this size ramp inside the swift file and can do it in JS instead.

Reply:
I don't think sizeThatFits is applicable here. The problem is two layout systems: UIKit and Yoga/flexbox

I need the sizes because I need to hardcode the `width / length` of the react native view using flexbox (how the web / React Native does layout). My solution previously was to pass the size from native so that I could set the width and height in React Native. There's probably a better, more complex solution to layout involving subclassing RCTShadowView (a class that React Native internally uses to layout views). I haven't investigated that.

Reply:
I see. Just to be clearer on the approach I meant with sizeThatFits:
After you call sizeThatFits, shouldn't be able to pass the dimensions to the React Native layer since you'll have the view in the right size?

Reply:
Are the numbers I have for each enum not the correct height/width (including rings)? Should I instead be instantiating an Avatar at each size, calling sizeThatFits, and passing the height/width from there?

Reply:
These are the sizes of the avatar (without considering the ring/gaps/presence if available).
`size: [ xSmall: 16pt, small: 24pt, medium: 32pt, large: 40pt, xlarge: 52pt, xxlarge: 72pt ]`

We do intent to change the frame on the avatar to be fixed (to the largest possible for the size enum specified).